template/build/webpack.prod.conf.js

```diff
@@ -120,7 +75,32 @@ const webpackConfig = merge(baseWebpackConfig, {
         ignore: ['.*']
       }
     ])
-  ]
+  ],
+  optimization: {
+    concatenateModules: true,
```

Comment:
In  `mode: 'production'`, it default enable.

Reply:
👍